src/sas/qtgui/Utilities/AddMultEditor.py

```diff
@@ -108,17 +109,28 @@ def setupModels(self):
         else:
             self.cbModel2.setCurrentIndex(0)
 
-    def readModels(self):
-        """ Generate list of models """
-        models = load_standard_models()
+    def readStandardModels(self):
+        """ Generate list of standard models """
+        s_models = load_standard_models()
         models_dict = {}
-        for model in models:
+        for model in s_models:
             if model.category is None or 'custom' in model.category:
                 continue
             models_dict[model.name] = model
 
         return sorted([model_name for model_name in models_dict])
 
```

Comment:
Should _readModels_ load standard models or all models?  Rather stylistic comment but _readModels_  and _readStandardModels_ seem to be quire redundant maybe we can remove some code duplication by passing arguments?